Frontend/src/App.tsx

```diff
@@ -12,23 +14,29 @@ import ReportModal from './components/reportModal/reportModalConnected'
 import './App.scss'
 
 const App: React.FC = () => {
+  const { init, UseGTMHookProvider } = useGTM()
+
+  useEffect(() => init({ id: 'GTM-N5LZSMG' }), [])
```

Comment:
@boriscirok dává ti smysl dát GTM id do .env souboru?

Reply:
Určitě, souhlas.